llama_cpp/server/app.py

```diff
@@ -820,16 +591,16 @@ class CreateChatCompletionRequest(BaseModel):
 async def create_chat_completion(
     request: Request,
     body: CreateChatCompletionRequest,
-    llama: llama_cpp.Llama = Depends(get_llama),
-    settings: Settings = Depends(get_settings),
+    llama: Llama = Depends(get_llama),
```

Comment:
Should fix the type signature here right? Since this depends now yields an `Optional[LlamaProxy]`